akka-docs/src/main/paradox/stream/stream-substream.md

```diff
@@ -15,7 +15,7 @@ To use Akka Streams, add the module to your project:
 
 ## Introduction
 
-Substreams are represented as `SubSource` or `SubFlow` instances, on which you can multiplex a single `Source` or `Flow`
+Substreams are represented as `SubSource` or @apidoc[akka.stream.*.SubFlow] instances, on which you can multiplex a single @apidoc[akka.stream.*.Source] or @apidoc[akka.stream.*.Flow]
```

Comment:
There is not `scaladsl.SubSource` in the akka. Shoud I use here only 'javadsl.SubSource'?

Reply:
AFAICS we don't have `SubSource` in any of the public Scala API's, so perhaps the best thing would be to only show it when looking at the Java version of the docs?